backends/tofino/bf-p4c/arch/add_t2na_meta.cpp

```diff
@@ -16,7 +16,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include "bf-p4c/arch/add_t2na_meta.h"
+#include "backends/tofino/bf-p4c/arch/add_t2na_meta.h"
```

Comment:
This should probably be just `#include "add_t2na_meta.h"` as it is in the same directory.  No need for a full path.

Lots of other places this could be cleaned up too -- possibly a separate cleanup change?

Reply:
I guess this is a stylistic choice. Not many guides mention it but Google style guide recommends to qualify all paths: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

I guess to avoid accidental conflicts with includes? 

I do not have a strong preference but the Tofino back end is not the only part which doesn't follow this style. So this likely should be a separate cleanup change.